uv.lock

```diff
@@ -406,7 +406,7 @@ wheels = [
 
 [[package]]
 name = "funding-service-design-utils"
-version = "5.1.6"
+version = "5.1.8"
```

Comment:
hmm this feels out of sync with pyproject.toml - possibly we need to update the version-bumping github action..?

Reply:
This is getting generated *from* the `pyproject.toml` value when you run `uv sync`. So could update the bump action to re-run `uv sync`?

Reply:
Will do a separate PR for this change